src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        final int MAX_TASK = 100;
+        Task[] tasks = new Task[MAX_TASK];
+        tasks[0] = new Task("Init");
+
+        printWelcomeMessage();
+
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
+        // Extract first part of command for action
+        String command = line.split(" ")[0];
+        while (!command.equals("bye")) {
+            switch (command) {
+            case "list":
+                printList(tasks);
+                break;
+            case "unmark":
+                unmarkTask(line, tasks);
+                break;
+            case "mark":
+                markTask(line, tasks);
+                break;
+            case "todo":
+                addToDo(line, tasks);
+                break;
+            case "deadline":
+                addDeadline(line, tasks);
+                break;
+            case "event":
+                addEvent(line, tasks);
+                break;
+            default:
+                addTask(line, tasks);
+                break;
+            }
+            line = in.nextLine();
+            command = line.split(" ")[0];
+        }
+
+        printBye();
+    }
+
+    private static void printWelcomeMessage() {
+        String logo = "_______  _______ _________ _______ _________ _______  _\n"
+                    + "(  ____ )(  ___  )\\__   __/(  ___  )\\__   __/(  ___  )( (    /|\n"
+                    + "| (    )|| (   ) |   ) (   | (   ) |   ) (   | (   ) ||  \\  ( |\\\n"
+                    + "| (____)|| |   | |   | |   | (___) |   | |   | |   | ||   \\ | |\n"
+                    + "|  _____)| |   | |   | |   |  ___  |   | |   | |   | || (\\ \\) | \n"
+                    + "| (      | |   | |   | |   | (   ) |   | |   | |   | || | \\   | \n"
+                    + "| )      | (___) |   | |   | )   ( |   | |   | (___) || )  \\  | \n"
+                    + "|/       (_______)   )_(   |/     \\|   )_(   (_______)|/    )_) \n";
+
+        System.out.println("_____________________________________________\n"
```

Comment:
Since you use this line a lot, you could consider creating a static String to print this line instead of copying it all the time?